ct_optcon/examples/NLOC_generalConstrained.cpp

```diff
@@ -3,180 +3,269 @@
  *
  * This example shows how to use general constraints alongside NLOC and requires HPIPM to be installed
  * The unconstrained Riccati backward-pass is replaced by a high-performance interior-point
- * constrained linear-quadratic Optimal Control solver.
+ * constrained linear-quadratic Optimal Control solver (HPIPM).
  *
  */
 
 #include <ct/optcon/optcon.h>
 #include "exampleDir.h"
-#include "plotResultsOscillator.h"
+#include "plotResultsPointMass.h"
 
 using namespace ct::core;
 using namespace ct::optcon;
 
 
-/*get the state and control input dimension of the oscillator. Since we're dealing with a simple oscillator,
- the state and control dimensions will be state_dim = 2, and control_dim = 1. */
-static const size_t state_dim = ct::core::SecondOrderSystem::STATE_DIM;
-static const size_t control_dim = ct::core::SecondOrderSystem::CONTROL_DIM;
+/* state and control input dimension of the point mass. States: x, y, v_x, v_y.
+ * Controls: linear velocity, angular velocity */
```

Comment:
Aren't both controls linear velocity?

Reply:
Actually its forces acting in x- and y-direction... I was still in my differential drive example when I wrote this